pages/common/dd.md

```diff
@@ -19,6 +19,14 @@
 
 `dd if=/dev/zero of={{file_1GB}} bs=1024 count=1000000`
 
+- Generate a system backup into a img file and show the progress:
+
+`dd if=/dev/{{drive_device}} of={{/path/to/file.img}} status=progress`
+
+- Flash an img file to a drive and show the progress:
```

Comment:
should we be consisted with the wording of the above command. Thoughts ?
```suggestion
- Restore a drive from an img file and show the progress:
```

Reply:
But that implies that the drives is being done again

Reply:
I think `restore` is better.